ui/imports/shared/panels/EditCroppedImagePanel.qml

```diff
@@ -164,7 +167,7 @@ Item {
                 id: addButton
 
                 icon.name: "add"
-
+                visible: !dropArea.containsDrag
```

Comment:
`dropArea` component has been removed. There is also reference of it in line `157` and `161`. States are not needed now!